external/opal-prd/hostboot-interface.h

```diff
@@ -71,6 +84,61 @@ enum MemoryError_t
 #define HBRT_PM_LOAD    0
 #define HBRT_PM_RELOAD  1
 
+/** Common return codes for scom_read(), scom_write(). */
+#define HBRT_RC_RANGE__scom   0x1000
+
+/* RC for a piberr is equal to 0x1000 plus the pib error value,
+ made into a negative */
+#define HBRT_RC_PIBERR_MASK            (0x00000000u - 0x00001007u) /* 0xFFFF_EFF9 */
+
```

Comment:
Why not simply add 0xFFFFEFF9.. that will make it easy to read.

Reply:
Because the negative number hex value loses all context.  The inverted (non-negative) value has real meaning : the first byte (10) indicates it is a scom rc, the second byte (07) is used as a mask to identify it was a pib error.  I find it easier to understand this way, though I acknowledge the hex is useful so I put it into the comment.

Reply:
Ok thanks for clarification.